Earthfile

```diff
@@ -51,6 +51,7 @@ e2e:
   ARG GOARCH=${TARGETARCH}
   ARG GOOS=${TARGETOS}
   ARG FLAGS="-test-suite=base"
+  ARG GOTESTSUM_FORMAT="standard-verbose"
```

Comment:
nice, i also like that by default when running e2e tests locally we'll get the more realtime streaming logging output 👍